assets/css/post-single.css

```diff
@@ -210,7 +210,7 @@
     display: block;
     margin: auto 0;
     padding: 10px;
-    color: rgba(255, 255, 255, .8);
+    color: var(--primary);
```

Comment:
This breaks the default text color of code blocks with the site's light theme. It's almost black on black.

**After:**
![screenshot from example site after change](https://user-images.githubusercontent.com/63574107/112741551-b0623280-8f3b-11eb-9c0d-c68004e48589.png)

**Before:**
![screenshot of example site before change](https://user-images.githubusercontent.com/63574107/112741562-d12a8800-8f3b-11eb-8ad9-848717f45262.png)

Reply:
Damn, yes, of course it is 🙈 My oversight was that dark/light for the _page_ is separate from dark/light for the _code_, and you can combine both, more or less arbitrarily. (Example:[ I change the code-bg to match the page-bg](https://www.jannikarndt.de/blog/2021/03/using_scalap_javap_and_the_scala_compiler/), which is why the change works _for me_). 
A generic "fix" would need a dedicated "code-text-color" variable in the `theme-vars.css`, so that it can be set for light and dark, together with the `--hljs-bg`, but separate from `--primary`.

I'm not sure if the use case justifies the additional complexity, or if it makes more sense to overwrite that _per user_. 

In any case, I don't have the testing experience to make this work, so I'll close the pr. Thank you for checking! 🙏

Reply:
I see what you're trying to do now. Would it work to just include a line like the following at the top of your `syntax.css` file?
```css
.chroma {
    color: /* your color here */;
}
```
This should then be overridden by the later definitions for specific classes.

Reply:
Hm, for _some_ cases, yes. But not if the code-background changes with light/dark as well. Also that file is generated by `hugo`, so I wouldn't change it but rather do all customisation inside the `custom.css`.